packages/perseus/src/widgets/interactive-graphs/graphs/utils.ts

```diff
@@ -1,5 +1,35 @@
+import {vec} from "mafs";
+
 import type {CollinearTuple} from "../../../perseus-types";
-import type {Interval, vec} from "mafs";
+import type {Interval} from "mafs";
+
+export function calculateAngleInDegrees([x, y]: vec.Vector2) {
+    return (Math.atan2(y, x) * 180) / Math.PI;
+}
+
+/**
+ * Given two points, generate the tip and rotation angle that an arrowhead should use when drawn in the graph.
+ * @param referencePoint - The point of the associated line used for reference to help determine arrowhead location.
+ * @param throughPoint - A point that the associated line passes through. Must be different from referencePoint.
+ * @param box - The box with which to intersect the line, in the form [[xMin, xMax], [yMin, yMax]]
+ */
+export const getArrowheadValues = (
+    referencePoint: vec.Vector2,
+    throughPoint: vec.Vector2,
+    box: [x: Interval, y: Interval],
+): {
+    tip: vec.Vector2;
+    angle: number;
+} => {
+    const extendedPoint = getIntersectionOfRayWithBox(
+        throughPoint,
+        referencePoint,
+        box,
+    );
+    const direction = vec.sub(extendedPoint, referencePoint);
```

Comment:
Do we need the `extendedPoint` to calculate the direction? Or is it equally correct to use `throughPoint`?

I'd prefer using `throughPoint` if we can, because that makes it clear that the direction does not depend on the extendedPoint calculation.

Reply:
Refactoring makes this issue moot.